src/async_impl/client.rs

```diff
@@ -103,6 +103,7 @@ struct Config {
     #[cfg(feature = "__tls")]
     tls: TlsBackend,
     http_version_pref: HttpVersionPref,
+    allow_http09_responses: bool,
```

Comment:
I'd rename this to just `http09_responses`, as the field and the method name.

Reply:
Changed it, assuming you didn't mean the actual method on the builder. If not tell me and I will change it on everything

Reply:
Oh sorry, yes I meant the method name too. It follows the pattern of naming protocol-specific properties with a prefix of that protocol.

Reply:
Fixed it!